### PR TITLE
Sign In

### PR DESCRIPTION
**Pull request by juanRodriguez17**
#### Description
* API url changed
* Sign in to the app with the respective backend flow
* Sign in screen according to designs

---

#### Preview
* image

### CODE REVIEW

File: app/src/main/res/drawable/edittext_backgorund_error.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="@android:color/transparent" />
+    <stroke
+        android:width="1dp"
+        android:color="@color/alizarin" />
+</shape>
```
Comment (ximenaperez) [inline app/src/main/res/drawable/edittext_backgorund_error.xml]: add an extra line at the end of the file

**Review by ximenaperez — CHANGES_REQUESTED** [review]
nice work! it seems the image is cut off on the screenshot though

File: app/src/main/res/drawable/edittext_background.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="@android:color/transparent" />
+    <stroke
+        android:width="1dp"
+        android:color="@color/black" />
+</shape>
```
Comment (ximenaperez) [inline app/src/main/res/drawable/edittext_background.xml:7]: same here

**Review by amaury901130 — APPROVED** [review]
Nice, please check this design, I think this shape should be a circle.
Screen Shot 2021-03-12 at 15 47 24

File: app/src/main/java/com/rootstrap/android/ui/activity/main/SignInActivity.kt [hunk 3]
```diff
@@ -31,11 +33,13 @@ class SignInActivity : PermissionActivity(), AuthView {
         Analytics.track(PageEvents.visit(VISIT_SIGN_IN))
 
         binding.signInButton.setOnClickListener { signIn() }
+        binding.signUpButton.setOnClickListener { signOut() }
+        binding.emailEditText.setOnFocusChangeListener { _, _ -> changeBackground(binding.emailEditText) }
+        binding.passwordEditText.setOnFocusChangeListener { _, _ -> changeBackground(binding.passwordEditText) }
```
Comment (amaury901130) [inline app/src/main/java/com/rootstrap/android/ui/activity/main/SignInActivity.kt]: please use `binding.apply{.....}`

Comment (juanRodriguez17) [reply app/src/main/java/com/rootstrap/android/ui/activity/main/SignInActivity.kt]: It has more sense, changed! Thanks

File: app/src/main/res/drawable/edittext_backgorund_error.xml (hunk 1 above)
Comment (juanRodriguez17) [reply app/src/main/res/drawable/edittext_backgorund_error.xml]: Done! Also the image `scaleType` property was changed in order to fix the cut off problem

**Review by ximenaperez — APPROVED** [review]

**Pull request merged**